fpga/src/main/scala/arty/System.scala

```diff
@@ -31,6 +31,12 @@ class E300ArtyDevKitSystem(implicit p: Parameters) extends RocketSubsystem
     with HasPeripheryGPIO
     with HasPeripheryPWM
     with HasPeripheryI2C {
+  val bootROM  = p(BootROMLocated(location)).map { BootROM.attach(_, this, CBUS) }
+  val maskROMs = p(MaskROMLocated(location)).map { MaskROM.attach(_, this, CBUS) }
+
+  val maskROMResetVectorSourceNode = BundleBridgeSource[UInt]()
+  tileResetVectorNexusNode := maskROMResetVectorSourceNode
```

Comment:
It seems weird to support both `bootROM` and `maskROM` above but then not do some connection here.
Do we require a `maskROM` but optionally allow a `bootROM` that never can be the reset location?

Reply:
Yea. This was a hack to try and do what the original code wanted. I need to look into this.